src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -406,7 +407,7 @@ RoverPositionControl::run()
 		bool manual_mode = _control_mode.flag_control_manual_enabled;
 
 		/* only run controller if position changed */
-		if (fds[0].revents & POLLIN) {
+		if (fds[0].revents & POLLIN || fds[4].revents & POLLIN) {
```

Comment:
I think we can use just local_position (`fds[4]`) and remove `fds[0]`

Reply:
Ha, pushing my very average git skills with this...never tried squashign commits before, I'll google and give it a try...

Reply:
Is local position also available if only using GPS?